src/components/entry/EntryTabs.js

```diff
@@ -32,24 +34,28 @@ export const TextContent = styled.article`
   }
 `
 
-const EntryTabs = () => (
-  <Page>
-    <EntryTab>
-      <TabList>
-        {/* TODO: Use Routing */}
-        <Tab>Overview</Tab>
-        <Tab>Reviews</Tab>
-      </TabList>
-      <TabPanels>
-        <TabPanel>
-          <EntryOverview />
-        </TabPanel>
-        <TabPanel>
-          <EntryReviews />
-        </TabPanel>
-      </TabPanels>
-    </EntryTab>
-  </Page>
-)
+const EntryTabs = () => {
+  const { pathname } = useLocation()
+
+  return (
+    <Page isDrawer={pathname.split('/')[1] === 'map'}>
```

Comment:
I think it's slightly better to pass `isDrawer` in as a prop from a routing-aware component (MobileLayout).

Reply:
Also, I think you should use `matchPath` for the condition. See my comments in `useRoutedTabs`